support-modules/acquisition-events/src/main/scala/com/gu/support/acquisitions/bigquery/BigQueryService.scala

```diff
@@ -103,6 +83,23 @@ class BigQueryService(config: BigQueryConfig) {
 }
 
 object BigQueryService {
+
```

Comment:
There are now 2 ways of constructing a `BigQueryService`.
The legacy way takes the `BigQueryConfig`, which contains secrets.
The new way takes a String containing json with just the credentials required for federated auth with Google Cloud.